knowledge/Users/klawrenc/.local/share/instructlab/taxonomy/knowledge/films/history/attribution.txt

```diff
@@ -0,0 +1,5 @@
+Title of work: History of films
+Link to work: https://www.blackwellpublishing.com/content/BPL_Images/Content_store/ThumbNail/BURGOYNE9781405146029/9781405146029.pdf
+Revision: https://en.wikipedia.org/wiki/History_of_film_technology
```

Comment:
The `revision` field should be a pointer to a specific version of the `link to work` URL, such as the versioned link of a Wikipedia document. So, if this link was the `link to work` (the reference), the `revision` field might have https://en.wikipedia.org/w/index.php?title=History_of_film_technology&oldid=1262139683, which can be found from the **Tools** dropdown in the upper right of a Wikipedia page as the **Permanent Link**.

If the source material does not have versioning, then it does not need to have this field.

Reply:
Note that, if you're using multiple sources, you would use multiple `link to work` fields, as in this example: https://github.com/instructlab/taxonomy/blob/38e3e3e88e3fd5acdbf32fd19fa5ae81178c9379/compositional_skills/grounded/linguistics/inclusion/attribution.txt